src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
 public class Application {
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            Ball randomBall = lottery.getRandomBall();
+            System.out.println(randomBall.toString());
```

Comment:
```suggestion
            System.out.println(randomBall);
```
toString() is redundant here